src/deadline/job_attachments/_diff.py

```diff
@@ -117,47 +118,184 @@ def compare_manifest(
+        root_relative_paths.append(root_relative_path)
+
+        return_path = select_path(
+            full_path=local_file,
+            relative_path=root_relative_path,
+            return_root_relative_path=return_root_relative_path,
+        )
         if root_relative_path not in input_files_map:
             # This is a new file
             logger.echo(f"Found difference at: {root_relative_path}, Status: FileStatus.NEW")
-            changed_paths.append(local_file)
+            changed_paths.append((return_path, FileStatus.NEW))
         else:
             # This is a modified file, compare with manifest relative timestamp.
             input_file = input_files_map[root_relative_path]
             # Check file size first as it is easier to test. Usually modified files will also have size diff.
             if file_stat.st_size != input_file.size:
-                changed_paths.append(local_file)
+                changed_paths.append((return_path, FileStatus.MODIFIED))
                 logger.echo(
                     f"Found size difference at: {root_relative_path}, Status: FileStatus.MODIFIED"
                 )
             elif int(file_stat.st_mtime_ns // 1000) != input_file.mtime:
-                changed_paths.append(local_file)
+                changed_paths.append((return_path, FileStatus.MODIFIED))
                 logger.echo(
                     f"Found time difference at: {root_relative_path}, Status: FileStatus.MODIFIED"
                 )
+
+    # Find deleted files. Manifest store files in relative form.
+    for manifest_file_path in diff_manifest.paths:
+        if manifest_file_path.path not in root_relative_paths:
+            full_path = os.path.join(root, manifest_file_path.path)
+            return_path = select_path(
+                full_path=full_path,
+                relative_path=manifest_file_path.path,
+                return_root_relative_path=return_root_relative_path,
+            )
+            changed_paths.append((return_path, FileStatus.DELETED))
     return changed_paths
+
+
+def pretty_print_cli(root: str, all_files: List[str], manifest_diff: ManifestDiff):
+    """
+    Prints to command line a formatted file tree structure with corresponding file statuses
+    """
+
+    # ASCII characters for the tree structure
+    PIPE = "│"
+    HORIZONTAL = "──"
+    ELBOW = "└"
+    TEE = "├"
+    SPACE = "    "
+
+    # ANSI escape sequences for colors
+    COLORS = {
+        "MODIFIED": "\033[93m",  # yellow
+        "NEW": "\033[92m",  # green
+        "DELETED": "\033[91m",  # red
+        "UNCHANGED": "\033[90m",  # grey
+        "RESET": "\033[0m",  # base color
+        "DIRECTORY": "\033[80m",  # grey
+    }
+
+    # Tooltips:
+    TOOLTIPS = {
+        FileStatus.NEW: " +",  # added files
+        FileStatus.DELETED: " -",  # deleted files
+        FileStatus.MODIFIED: " M",  # modified files
+        FileStatus.UNCHANGED: "",  # unchanged files
+    }
```

Comment:
Could we model file status related configurations better?

Some idea - https://stackoverflow.com/questions/59916345/adding-a-property-to-an-enum

Reply:
I agree- let me look into this refactor after we complete all the CLIs and name cleanups.